File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Chữ số hàng trăm/NHAPXUAT4.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    int count = 0;
    int temp, res;
    while(n != 0)
    {
        temp = n % 10;
        count ++;
        if(count == 3)
            res = temp;
    }
    if(count < 3)
        cout << "-1";
    else
        cout << res;
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Chữ số hàng trăm/NHAPXUAT4.py
a = int(input())
count = 0
while a != 0:
    temp = a % 10
    count += 1
    if count == 3:
        res = temp
    a //= 10
if count < 3:
    print(-1, end="")
else:
    print(res, end="")